README.md

```diff
@@ -77,10 +77,18 @@ SELECT min_by(x, y) FROM VALUES (1, 10), (2, 5), (3, 15), (4, 8) as tab(x, y);
 -- | 2                   |
```

Comment:
It seems we have enough example, just a description in "Done" section, will be fine.